nixos/modules/services/printing/cupsd.nix

```diff
@@ -4,7 +4,7 @@ with lib;
 
 let
 
-  inherit (pkgs) cups cups-pk-helper cups-filters;
+  inherit (pkgs) cups cups-pk-helper cups-filters xdg-open;
```

Comment:
`xdg-open` is in `pkgs.xdg-utils`. See also `nix-locate`, that tool should help to answer those questions.

Reply:
Thank you, fixed :)